package/obs/rmt-server.spec

```diff
@@ -320,8 +321,12 @@ getent passwd %{rmt_user} >/dev/null || \
 
 %post
 %service_add_post rmt-server.target rmt-server.service rmt-server-migration.service rmt-server-mirror.service rmt-server-sync.service rmt-server-systems-scc-sync.service
-cd %{_datadir}/rmt && bin/rails rmt:secrets:create_encryption_key >/dev/null RAILS_ENV=production && \
-cd %{_datadir}/rmt && bin/rails rmt:secrets:create_secret_key_base >/dev/null RAILS_ENV=production && \
+
+# Rails by default creates `secrets.yml.key` with `0600` file mode, see here
+# https://github.com/rails/rails/blob/6-0-stable/railties/lib/rails/generators/rails/encryption_key_file/encryption_key_file_generator.rb
+cd %{_datadir}/rmt && runuser -u root -g %{rmt_group} -- bin/rails rmt:secrets:create_encryption_key >/dev/null RAILS_ENV=production && \
```

Comment:
is this save to assume `runuser` is existing? I mean does all environments come with pam installed. I think so but I'm not sure.

Reply:
I think the package is a dependency. And this has been used in the spec file before 2.10. I'm not introducing a new dependency